### PR TITLE
margin-correction

### LINKED ISSUE #7868: Kanban columns margin no longer extend to the bottom
## Bug Description
As per title

Screenshot 2024-10-20 at 00 36 04

## Expected behavior

Column margins should extend to the bottom.

Screenshot 2024-10-20 at 00 40 31

https://www.figma.com/design/xt8O9mFeLl46C5InWwoMrN/Twenty?node-id=24955-194300&node-type=frame&t=IlVBRHG7XrFQOD7c-0

### PR DESCRIPTION
**Pull request by ayesparshh**
fixes #7868

### CODE REVIEW

File: packages/twenty-front/src/index.css
Comment (ehconitin) [inline packages/twenty-front/src/index.css:1]: I am not sure.
@lucasbordeau wdyt

**Comment by ehconitin** [comment]
@ayesparshh isn't this issue assigned to someone else? 
[Contributing guidelines](https://github.com/twentyhq/twenty/blob/4bd8945519ed4b0e87beb27fec2a67c4de06c4ba/.github/CONTRIBUTING.md)

**Comment by charlesBochet** [comment]
Closing this one as we wait for @VAIBHAVSING, happy to re-open it if we don't get update from @VAIBHAVSING

**Comment by charlesBochet** [comment]
@ayesparshh thanks for the PR, any help is much appreciated. Feel free to work on any other unassigned issue :)

**Comment by ayesparshh** [comment]
@charlesBochet I guess no pr raised against this issue should I work again on this?

**Comment by charlesBochet** [comment]
@ayesparshh I cannot reproduce anymore actually, can you?

**Comment by charlesBochet** [comment]
Actually, I can still reproduce! Sure, would help if you could tackle it, thanks @ayesparshh!

**Pull request closed without merging**